src/containers/Contact/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { useForm } from '../../hooks/useForm';
+import { isRequired, isEmail, maxChars } from '../../hooks/useForm/validators';
+
+const Contact: React.FC = () => {
+  const formInputs = {
+    name: {
+      value: '',
+      validators: [isRequired],
+    },
+    email: {
+      value: '',
+      validators: [isRequired, isEmail],
+    },
+    comment: {
+      value: '',
+      validators: [maxChars(20)],
+    },
+  };
+
+  const { values, onChange, errors, disableSubmit } = useForm(formInputs);
+
+  const onSubmitHandler = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+  };
+
+  return (
+    <React.Fragment>
+      <h2>Contact</h2>
+      <form onSubmit={onSubmitHandler}>
+        {/* Name field*/}
+        <label>Name: </label>
+        <br />
+        <input
+          type="text"
+          name="name"
+          onChange={onChange}
+          value={values.name}
```

Comment:
@Stefafrontani where are u seeing `value.value` in the implementation it gets like `values.fieldName` like `name` in this scenario.

Reply:
On the line the comment was created. Not here.